### PR TITLE
Add test 'Check number of orders in order history page'

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.7.x
| Description?      | Add test 'Check number of orders in order history page'
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | `TEST_PATH="functional/FO/03_userAccount/04_orderList.js" URL_FO=shopUrl/ npm run specific-test`
| Possible impacts? | none

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/22874)

### CODE REVIEW

**Review by nesrineabdmouleh — APPROVED** [review]

File: tests/UI/campaigns/functional/FO/03_userAccount/04_orderList.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+  it('should go to order history page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToOrderHistoryPage', baseContext);
+
+    await foMyAccountPage.goToHistoryAndDetailsPage(page);
+    const pageHeaderTitle = await foOrderHistoryPage.getPageTitle(page);
+    await expect(pageHeaderTitle).to.equal(foOrderHistoryPage.pageTitle);
+  });
+
+  it('should go to order history page', async function () {
```
Comment (NeOMakinG) [inline tests/UI/campaigns/functional/FO/03_userAccount/04_orderList.js:75]:
```suggestion
  it('should see order listing', async function () {
```

Comment (NeOMakinG) [reply tests/UI/campaigns/functional/FO/03_userAccount/04_orderList.js:75]: Not sure about the string, but you forgot to change it anyway

Comment (boubkerbribri) [reply tests/UI/campaigns/functional/FO/03_userAccount/04_orderList.js:75]: It's the right string, no need to change it

**Review by PierreRambaud — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @boubkerbribri

**Pull request merged**